### PR TITLE
WIP: New pipeline and workflow

### PR DESCRIPTION
**Pull request by pashcovich**
New pipeline and workflow for deploy

### CODE REVIEW

File: .github/workflows/buildAndDeployStage.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+jobs:
+  buildAndDeploy2Stage:
+    name: "Build and Deploy to stage"
+    runs-on: self-hosted
+    env:
+      WERF_SECRET_KEY: ${{ secrets.WERF_SECRET_KEY }}
+      ENV: "stage"
+    steps:
+      - uses: actions/checkout@v1
```
Comment (chekalsky) [inline .github/workflows/buildAndDeployStage.yml:15]:
```suggestion
      - uses: actions/checkout@v2
```

File: .github/workflows/buildAndDeployDev.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+jobs:
+  buildAndDeploy2Dev:
+    name: "Build and Deploy to dev"
+    runs-on: self-hosted
+    env:
+      WERF_SECRET_KEY: ${{ secrets.WERF_SECRET_KEY }}
+      ENV: "dev"
+    steps:
+      - uses: actions/checkout@v1
```
Comment (chekalsky) [inline .github/workflows/buildAndDeployDev.yml:17]:
```suggestion
      - uses: actions/checkout@v2
```

File: .github/workflows/buildAndDeployStage.yml (hunk 1 above)
Comment (pashcovich) [reply .github/workflows/buildAndDeployStage.yml:15]: No, it was done on purpose

File: .github/workflows/buildAndDeployDev.yml (hunk 2 above)
Comment (pashcovich) [reply .github/workflows/buildAndDeployDev.yml:17]: No, it was done on purpose

**Review by chekalsky — APPROVED** [review]

**Review by xinger — APPROVED** [review]

**Review by ekorolev — APPROVED** [review]

**Pull request merged**